content/flux/v0.x/stdlib/experimental/bitwise/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Flux experimental bitwise package
+list_title: bitwise package
+description: >
+  The Flux experimental `bitwise` package provides functions for performing
+  bitwise operations on integers.
+  Import the `experimental/bitwise` package.
+menu:
+  flux_0_x_ref:
+    name: bitwise
+    parent: experimental
+weight: 301
+flux/v0.x/tags: [functions, bitwise, package]
+cascade:
+  introduced: 0.138.0
+---
+
+The Flux experimental `bitwise` package provides functions for performing bitwise
+operations on integers and unsigned integers.
+Import the `experimental/bitwise` package:
+
+```js
+import "experimental/bitwise"
+```
+
+## Functions
+Functions prefixed with `s` operate on [signed integers (int)](/flux/v0.x/data-types/basic/int).
```

Comment:
would it make sense to link to the operate part?: /flux/v0.x/data-types/basic/int/#operate-on-integers

Reply:
Good callout. I should add examples for performing bitwise operations with integers.